modified VAR_DIR to include instance name

jszigetvari opened the pull request:
contrib/solaris-packaging/syslog-ng@default: modified some default values for parameters

    * minor improvements to fix possible file collisions across instances
    * VAR_DIR will now contain the instance name for non-default instances
      if not defined otherwise
    * persist, pid and ctl files will not reflect the instance names, but
      their paths will
    * changed VAR_DIR paths

Signed-off-by: Janos SZIGETVARI <jszigetvari@gmail.com>

kira-syslogng commented:
This user does not have permission to start the build. Can one of the admins verify this patch and start the build? (admin please type: ok to test)

MrAnno commented:
@kira-syslogng ok to test

kira-syslogng commented:
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

bazsi commented:
This looks good to me, but I can't test it myself. I can assume that you tested it yourself, right?

jszigetvari commented:
> I can assume that you tested it yourself, right?

As I don't have a Solaris 10-11 test machine for OSE, I haven't . The original version was implemented for PE, I removed the PE-specific parts, and added OSE-specific ones. I did the testing with PE only.
Now I will try to build OSE on Solaris 11 for the sake of testing it.

kira-syslogng commented:
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

MrAnno commented:
Thanks.
